presto-main/src/main/java/com/facebook/presto/type/TimeWithTimeZoneOperators.java

```diff
@@ -114,6 +117,18 @@ public static Slice castToSlice(@SqlType(StandardTypes.TIME_WITH_TIME_ZONE) long
         return utf8Slice(printTimeWithTimeZone(value));
     }
 
+    @ScalarOperator(CAST)
+    @SqlType(StandardTypes.TIME_WITH_TIME_ZONE)
+    public static long castFromSlice(ConnectorSession session, @SqlType(StandardTypes.VARCHAR) Slice value)
```

Comment:
I am somewhat afraid that by adding new cast operators we can run into problems that some SQL queries which run correctly before change, no will start failing with ambiguity related error.

Is that possible?